dog_classifier/net/network.py

```diff
@@ -25,29 +36,81 @@ def DogNN():
 def DogNNv2():
     # K.set_image_dim_ordering('th')
     shape_input = (None, None, 3)
-
     model = Sequential()
-    model.add(Conv2D(filters=2, kernel_size=(3, 3), activation='relu',
-                     dilation_rate=(2, 2), input_shape=shape_input))
-    # model.add(Conv2D(filters=4, kernel_size=(7, 7), activation='relu'))
-    # model.add(Conv2D(filters=8, kernel_size=(7, 7), activation='relu'))
-    # model.add(Conv2D(filters=8, kernel_size=(7, 5), activation='relu'))
-    # model.add(Conv2D(filters=8, kernel_size=(5, 7), activation='relu'))
-    # model.add(Conv2D(filters=8, kernel_size=(5, 7), activation='relu'))
-    # model.add(Conv2D(filters=16, kernel_size=(5, 5), activation='relu'))
-    # model.add(Conv2D(filters=4, kernel_size=(3, 3), activation='relu'))
+    model.add(Conv2D(filters=4, kernel_size=(3, 3),
+                     dilation_rate=(2, 2),
```

Comment:
Warum benutzt du hier eine dilation rate? Und später auch bei den ersten Conv2d Filtern?

Reply:
Um eher allgemeine Features zu finden, aber war auch nur ein test. 
Hier ist noch eine interessante [Internetseite](https://towardsdatascience.com/understanding-2d-dilated-convolution-operation-with-examples-in-numpy-and-tensorflow-with-d376b3972b25) zum Thema